LDAR_Sim/src/file_processing/output_processing/output_utils.py

```diff
@@ -83,6 +84,29 @@ class TIMESERIES_COL_ACCESSORS:
     METH_DAILY_SURVEY_TIME = "{method} Survey Time (Minutes)"
 
 
+def percent_difference(a: float, b: float) -> float:
```

Comment:
Should this have error handling for div by 0?

Reply:
Will add in handling for the case both values are 0